packages/create-modular-react-app/src/index.ts

```diff
@@ -5,21 +5,19 @@ import * as path from 'path';
 import chalk from 'chalk';
 import * as semver from 'semver';
 
-function execSync(
-  file: string,
-  args: string[],
-  options: { log?: boolean } & execa.SyncOptions = { log: true },
-) {
-  const { log, ...opts } = options;
-  if (log) {
-    console.log(chalk.grey(`$ ${file} ${args.join(' ')}`));
+function exec(file: string, args: string[], cwd: string) {
+  console.log(chalk.grey(`$ ${file} ${args.join(' ')}`));
+  try {
+    return execa(file, args, {
+      stdin: process.stdin,
+      stderr: process.stderr,
+      stdout: process.stdout,
+      cwd,
+    });
+  } catch (e) {
+    console.error(chalk.red(`$ FAILED ${file} ${args.join(' ')}`));
```

Comment:
Doesn't console.error print in red by default?

Reply:
Nope

Reply:
Ah I was thinking of logger.error.